### PR TITLE
LPS-127750 Check required fields in all form views on populateConfigData

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

**Comment by naraandrade** [comment]
ci:reevaluate:1172004_7252

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:test:relevant

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js [hunk 1]
```diff
@@ -96,31 +96,27 @@ export function populateConfigData([
 		};
 	});
 
+	const dataObject = dataObjects.find(({id}) => id === app.dataDefinitionId);
```
Comment (rafaprax) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js:99]: Hey @naraandrade, Can you move this statement directly to where it is used?
`checkRequiredFields(formViews, dataObjects.find(({id}) => id === app.dataDefinitionId))`

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js (hunk 1 above)
Comment (rafaprax) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js:99]: Never mind @naraandrade. I didn't see the other usage in the config.

Sorry :)

**Review by rafaprax — APPROVED** [review]

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js (hunk 1 above)
Comment (rafaprax) [reply modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/actions.es.js:99]: So, using the _as used_ priciple we move it then.

Thx!

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**